core/templates/domain/skill/SkillDifficultyObjectFactorySpec.ts

```diff
@@ -45,5 +45,19 @@ describe('Skill Difficulty object factory', () => {
       };
       expect(skillDifficulty.toBackendDict()).toEqual(skillDifficultyDict);
     });
+
+    it('should set the description of the skill difficulty instance', () => {
+      var skillDifficulty =
+        skillDifficultyObjectFactory.create('1', 'test skill', 0.3);
```

Comment:
```suggestion
      var skillDifficulty = skillDifficultyObjectFactory.create(
        '1', 'test skill', 0.3);
```

OR 

```
      var skillDifficulty = (
        skillDifficultyObjectFactory.create('1', 'test skill', 0.3));
```

[Break line after parenthesis]